aks/application/variables.tf

```diff
@@ -92,3 +92,13 @@ variable "probe_command" {
   description = "Command for the liveness and startup probe"
 }
 
+variable "github_username" {
+  type        = string
+  default     = null
+  description = "Github user authorised to access the private registry"
+}
+variable "github_pat" {
```

Comment:
Could we call this something like like `github_personal_access_token` or `github_token` or `github_password`?

I didn't immediately know what "pat" stood for without needing to read the description.

Reply:
done!